### ISSUE TITLE
Revamp Fort Magma

### ISSUE THREAD

**Issue by Alayan-stk-2** [STK-helper/stk-code#21]
One of the more graphically outdated tracks, and with a rather boring path.

Is the final track of the game for the boss race, so high priority.

**Comment by qwertychouskie** [comment]
Music: https://jacobspctuneup.tk/STK/Music/fortress_stkr.ogg

**Comment by qwertychouskie** [comment]
Lava flow/volcano: https://samuncle.net/cloud/index.php/s/oCLjfdDTPTFLFXc

**Comment by Alayan-stk-2** [comment]
Yes, the lava flow is very nice and would work very well. Samuncle should investigate a way to remove the "pulsing", however, as that's quite noticeable.
